aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/SdkSettings.kt

```diff
@@ -17,6 +19,20 @@ class SdkSettings private constructor(private val awsSdk: ObjectNode?) {
             SdkSettings(coreRustSettings.customizationConfig?.getObjectMember("awsSdk")?.orNull())
     }
 
+    /** Path to the `sdk-default-configuration.json` config file */
+    val defaultsConfigPath: Path get() =
```

Comment:
should probably note in comments that this is unused currently

Reply:
I think this kind of comment is highly likely to get out of date and cause confusion.